### PR TITLE
MESH-834/Make the bridge easier to test

### PR DESCRIPTION
**Pull request by adamdossa**
Sets Alice as the bridge admin in `testnet_genesis`

Allows the bridge controller to submit `handle_bridge_tx` calls

### CODE REVIEW

**Comment by maxsam4** [comment]
It doesn't really matter but I'd like to point out that Alice's stash account has been added as the admin and not Alice herself.

File: pallets/runtime/src/bridge.rs [hunk 1]
```diff
@@ -311,7 +318,8 @@ decl_module! {
             DispatchResult
         {
             let sender = ensure_signed(origin)?;
-            ensure!(sender == Self::controller(), Error::<T>::BadCaller);
+            ensure!(sender == Self::controller() || sender == Self::admin(), Error::<T>::BadCaller);
```
Comment (satyamakgec) [inline pallets/runtime/src/bridge.rs]: isn't it better to add a TODO so we can remove the admin check at the production?

**Comment by adamdossa** [comment]
> It doesn't really matter but I'd like to point out that Alice's stash account has been added as the admin and not Alice herself.

fixed to Alice rather than her stash

File: pallets/runtime/src/bridge.rs (hunk 1 above)
Comment (adamdossa) [reply pallets/runtime/src/bridge.rs]: added todo comment

**Pull request merged**